products/find-va-forms-redesign/form-detail-landing-pages/WIP-technical-specs-and-docs/va-forms-cms-governance.md

```diff
@@ -1,27 +1,31 @@
 # VA Forms - CMS Governance
 
-## Stages
 
-### Stage 1
-- The Forms DB is the source of truth for general form information (name, location, etc.). The CMS is updated by nightly migrations from the Forms DB.
-- The CMS is the source of truth for additional information (description, online tool, etc.) used on the form detail pages.
-
-### Stage 2
-- The Forms DB is retired
-- The CMS is the source of truth for all form data.
+## Stage 1
+- The VA Forms DB is the source of truth for general form information (form name, form number, form owner/ID, etc.). 
+- The CMS is updated by nightly migrations from the VA Forms DB.
+- The CMS is the source of truth for the additional forms information (related to, online tool, related forms/instructions, etc.) used on the form detail pages.
 
 ## Governance in Stage 1
-### Ownership 
-The VSA Public Websites team will own the all of the VA Forms content. Ownerwhip will be assigned during migration. Only CMS users with Public Websites team membership will be able to edit this content.
+
+### Ownership
+- The VSA Public Websites team will 'own' all of the VA forms content that originate in the CMS. Ownerwhip will be assigned during migration. Only Public Websites CMS users will be able to edit this content.
+- Forms data that originate in the VA Forms DB (e.g., form number, form owner/ID, form PDF URL) will continue to be owned by the forms managers and managed in the source VA Forms DB.
 
 ### Editorial workflow within CMS
 Form pages will be governed by the same workflow as other content ('Draft', 'In review' 'Published').
 
-### Permissions
+### CMS permissions
 
-- Because basic form data will be maintained in the Forms DB, non-admin users will not have permission to create or delete Form content in the CMS.
-- Users with appropriate roles and administration membership will have permission to edit Forms pages, but will not be able to edit fields that come from Forms DB.
+- Because basic form data will be maintained in the VA Forms DB, CMS users will not have the ability to create or delete form detail landing pages in the CMS. [@beth - not sure if I understood what this means, so I took a stab based on what I think I understand and what I would expect./jen] 
```

Comment:
'Admin' is a special role, reserved mostly for developers. Admins can basically do anything, so these permissions don't apply to them. All other CMS users (non-admins) won't be able to create or delete Forms pages.

Reply:
We might need to talk this one out. Looks really critical. I want to make sure this is set right. Because sometimes even I get an error message saying I'm not an admin with access to a page, even though I am supposed to have global UX admin rights. 

1/ I've seen on the CMS permissions spreadsheet from Howie that there are "Administrators" and also "User Admins" -- are these different? 

2/ Why does anyone need to create a form detail page - that should be auto generated when a new form is added to the VA forms db, no? (Now in the future we may/will want to add a suppress feature for non-VA forms, but for now.) 

3/ For deleting a form detail page - this is tricky and has some downstream things that need to be considered - like redirects, potential for 404s, and current behavior of form detail landing pages (they are not currently automatically removed when a form is deleted from the DB).

Reply:
per convo - Content Publisher should be able to update; publish; and delete form detail landing pages. (This is a change from Administrator.) 

per convo - When forms are migrated into the CMS, it will automatically generate a form detail landing page in **draft** state. (This is a change in understanding from 'publish' state.)